Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -1398,6 +1398,67 @@ def visit_DefNode(self, node):
         node.decorators = None
         return self.chain_decorators(node, decs, node.name)
 
+    def visit_CFuncDefNode(self, node):
```

Comment:
I think it would be nice to reduce the code duplication here.

Reply:
removed it so the duplication is gone